tests/blackbox/EncryptControllerTests.cs

```diff
@@ -125,5 +125,22 @@ public async Task AnonymousRequestToDecryptEndpointShouldFail()
 
             Assert.Equal(HttpStatusCode.Unauthorized, result.StatusCode);
         }
+
+        [Fact]
+        public async Task BadRequestShouldFail()

```

Comment:
```suggestion
        public async Task Encrypt_BadRequest_ShouldFail()
```
Also, what about decrypt?